### ISSUE TITLE
Compiling error

### ISSUE THREAD

**Issue by JOYScka** [scout-app/scout-app#394]
I'm receiving a Error (0x01)
```
Invalid CSS after "...mplate-columns:": expected expression (e.g. 1px, bold), was "[full-start] minmax"
```

The offending code is 
```
grid-template-columns: [full-start] minmax(0, 1fr) / [main-start] repeat(4, minmax(0, 270px)) / [main-end] minmax(0, 1fr) / [full-end];
```

I wish I had more info and more knowledge to share. But I'm afraid I'm new to grid columns and this is the new core joomla4 template.

This is the only line causing an issue - when it is removed the sass is compiled. Any insights you could provide would be greatly appreciated - as I have tried in vain to set up sass through the terminal but I'm a bit of a noob and I'm totally relying on your awesome little app.

**Comment by TheJaredWilcurt** [comment]
Scout-App v2.12.12 uses LibSass v3.3. CSS Grid support wasn't added into LibSass until v3.5.

The next release of Scout-App will have support for this. If you would like to use the latest code now (rather than wait for the next release), you can follow [these instructions](https://github.com/scout-app/scout-app/blob/master/documentation/contributing.md#running-scout-app-locally-for-development).

**Comment by tonextone** [comment]
OK, I'm looking forward to the next release. Really. 
Thanks.

**Comment by TheJaredWilcurt** [comment]
@JOYScka @tonextone 

Closing issue. This has been released in **[Scout-App v2.18.16](https://github.com/scout-app/scout-app/releases/tag/v2.18.16)**.
